python/desc/twinkles/twinkles_sky.py

```diff
@@ -106,11 +103,11 @@ def __init__(self,
         # Lists of component phosim Instance Catalogs and CatalogDBObjects
         # Stars
         self.compoundStarDBList = [StarObj, CepheidStarObj]
-        self.compoundStarICList = [PhoSimCatalogPoint, PhoSimCatalogPoint]
+        self.compoundStarICList = [TwinklesPhoSimCatalogPoint, TwinklesPhoSimCatalogPoint]
```

Comment:
I was able to run this code after merging into `u/danielsf/cache_stars`, but there were a number of conflicts I had to iron out.  It seems that this PR, #373, and #381 all touch the same code.   Is there a preferred order to merge all of these into master?